common/logger/logger.go

```diff
@@ -30,6 +31,7 @@ import (
 
 var (
 	logger Logger
+	loggerRWMutex = sync.RWMutex{}
```

Comment:
In most cases of using nacos-sdk-go, users do not initialize multiple nacos clients in the program, and most users choose to override the logger of nacos-sdk-go.

Reply:
Currently, locks are not necessary in this PR, there is no guarantee that each client will use its own logger.

Reply:
Perhaps we could consider using Sync.once. To ensure that the client.CreateNamingClient() is executed successfully only Once.